doc/source/indexing.rst

```diff
@@ -1785,17 +1771,17 @@ interpreter executes this code:
 
 .. code-block:: python
 
-   dfmi.loc[:, ('one', 'second')] = value
+   dfmi.loc[:, ('one', 'second')] = value                          # noqa: F821
```

Comment:
what do you think of adding `value = 'z'` before this, so we don't need to have all the `F821`? Can be in a `:suppress:` block, or directly here, whatever you think it's better.